CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Refactor dwarfWalker::findConst ([1160](https://github.com/dyninst/dyninst/issues/1160))
 - Add readable name for Symtab::typeRef ([1157](https://github.com/dyninst/dyninst/issues/1157))
 - DwarfWalker: clean up interfaces for findDieName and findName ([1154](https://github.com/dyninst/dyninst/issues/1154))
+- Added automated docker build for development and testing
```

Comment:
As an aside, we usually only update the CHANGELOG on release, but this is ok.

Reply:
Noted -   I update mine as I go because it's easier to remember!